simd_emulation/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 The Rust verification tools Authors.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+# this rule comes first to make sure that it is the default target
+default: simd_emulation.bc
+
+RUSTC = rustc
```

Comment:
Use `?=` instead of `=`, so users can set it in their shell init script.

Reply:
Done